package.json

```diff
@@ -5,6 +5,10 @@
   "main": "lib/index.js",
   "module": "lib-esm/index.js",
   "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "(lib|lib-esm)/behaviors/focusZone.js",
+    "(lib|lib-esm)/behaviors/focusTrap.js"
```

Comment:
<img width="656" alt="Screen Shot 2021-07-17 at 10 31 19 AM" src="https://user-images.githubusercontent.com/8616962/126040274-b51495f3-6699-4f19-bc98-e05a0f293b0e.png">

This polyfill call is the side effect